ddl/job_table.go

```diff
@@ -582,14 +595,13 @@ func GetBackfillJobsForOneEle(sess *session, batch int, excludedJobIDs []int64,
 
 	var err error
 	var bJobs []*BackfillJob
-	s := newSession(sess)
-	err = runInTxn(s, func(se *session) error {
-		currTime, err := GetOracleTimeWithStartTS(s)
+	err = runInTxn(sess, func(se *session) error {
+		currTime, err := GetOracleTimeWithStartTS(se)
 		if err != nil {
 			return err
 		}
 
-		bJobs, err = GetBackfillJobs(sess, BackfillTable,
+		bJobs, err = GetBackfillJobs(se, BackfillTable,
```

Comment:
@zimulala I assume it should use the new session `se` from runInTxn instead of the outer `sess` here?